tests/p2p/test_protocol.py

```diff
@@ -329,11 +325,11 @@ def test_get_data(self):
         self.conn.run_one_step()  # TIPS
         self.assertIsConnected()
         missing_tx = '00000000228dfcd5dec1c9c6263f6430a5b4316bb9e3decb9441a6414bfd8697'
-        yield self._send_cmd(self.conn.proto1, 'GET-DATA', missing_tx)
```

Comment:
Why remove the yield?

Reply:
The `_send_cmd()` method does not return a `Deferred`, so the `yield` is not necessary.

Reply:
Ok. Thanks!